### ISSUE TITLE
At a certian point, Merge Duplicates shows a blank screen after the animation.

### ISSUE THREAD

**Issue by MattGyverLee** [sillsdev/TheCombine#1700]
**Describe the bug**

2 weeks ago, During our workshop, we used theCombine (the web version) to work through duplicates. Each of the 5 projects eventually came to a point where "Merge Duplicates" was enabled, but after the animation, they would be presented with a blank white screen. We assumed that this must mean that there were no more duplicates to show, and that it was simply failing to catch this state gracefully.

**To Reproduce**

1. Open Jimi2 under MatthewLee (these are no longer the primary data, as they have moved to FLEx/LanguageDepot, so the admins can play with it)
2. Try to Merge Duplicates
3. Get The Combine animation.
4. Get white screen.
5. Be confused with me.

**Expected behavior**

The tool should gracefully tell you that there are no more duplicates to handle.

**Environment:**

- OS: Windows 10
- Browser Version Chrome 103
- Most were using the French interface with the Browser/cookie hack in https://github.com/sillsdev/TheCombine/issues/1684

**Comment by imnasnainaec** [comment]
@MattGyverLee Thanks for documenting this bug. There is a non-blank screen that tells when no more potential duplicates are found, so this is definitely an issue for us to investigate. Can you add me (imnasnainaec) to the project in The Combine or send me (imnasnainaec@gmail.com) the lift files for the project in questions?

**Comment by MattGyverLee** [comment]
I added you to Jimi2. I just tested and got the blank white screen on my phone that I had seen on the desktop.

**Comment by imnasnainaec** [comment]
@MattGyverLee Thanks! That let me see the console error accompanying the bug:
```
Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at hu (react-dom.production.min.js:237:48)
    at Ta (react-dom.production.min.js:170:408)
    at Object.e [as onStateChange] (useSelector.js:78:7)
    at Object.o [as callback] (Subscription.js:90:20)
    at Subscription.js:19:20
    at wu (react-dom.production.min.js:244:160)
    at Object.notify (Subscription.js:15:7)
    at Object.notifyNestedSubs (Subscription.js:85:15)
    at Object.d [as onStateChange] (connectAdvanced.js:79:9)
    at Object.o [as callback] (Subscription.js:90:20)
```

**Comment by imnasnainaec** [comment]
If I export the project and import it in a new project, the merging tool works, so whatever the issue is with `Jimi-2` is not preserved in export. We'll keep digging into this.

**Comment by MattGyverLee** [comment]
All 5 of our projects eventually got the white screen after many cycles of merging (the team with the least entries (Jimi-2) got it after one day, the team with the most entries after 3 days). This is why I assumed that it was related to running out of entries to merge. Kangou-2 should be in the same situation.

**Comment by MattGyverLee** [comment]
When you get the white screen, the back button doesn't get you back to the main menu. It seems to be fatal to the React session. I had to go back to TheCombine.app to get around the error, often logging in again.

**Comment by imnasnainaec** [comment]
> When you get the white screen, the back button doesn't get you back to the main menu. It seems to be fatal to the React session. I had to go back to TheCombine.app to get around the error, often logging in again.

@MattGyverLee I recommend bookmarking https://thecombine.app/app to avoid having to log back in as often.

**Comment by jasonleenaylor** [comment]
> ```
> Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
> ```

Hmm, this minified error saved 8 characters... I wonder if they have minified any shorter error messages.

**Comment by imnasnainaec** [comment]
Dev note: The screen briefly flashes "Merge Set 1 of 2", so the problem appears to be in the MergeDups component and not in the goal router.

**Comment by imnasnainaec** [comment]
https://legacy.reactjs.org/docs/error-decoder.html/?invariant=185

> Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

**Comment by imnasnainaec** [comment]
I just started investigating this issue again. It still crashes on our live server (v1.1.3), however, on our QA server (v1.1.3-master.49) it doesn't crash, but does show a merge candidate with two words, one of which is empty (no vernacular or senses).
![Screenshot 2023-11-10 095912](https://github.com/sillsdev/TheCombine/assets/6411521/0522eff2-0c98-4cc3-8b7e-c5c1122e3ad7)

**Comment by imnasnainaec** [comment]
The project has somehow gained two words with no senses, which shouldn't happen. I'll work on safeguards against this.
![Screenshot 2023-11-10 100529](https://github.com/sillsdev/TheCombine/assets/6411521/7ca5fd20-0188-4698-9a33-a9e5290e0d4c)

**Comment by imnasnainaec** [comment]
@MattGyverLee This issue is fixed in our latest release (v1.1.5).
